openhtf/__init__.py

```diff
@@ -241,6 +251,8 @@ def execute(self, test_start=None):
       finally:
         self._executor = None
 
+      return (final_state.test_record.outcome == test_record.Outcome.PASS)
```

Comment:
this return is still inside the outermost `finally:`, should be de-dented one more level.